self-hosted/distributed-hypertables/create-distributed-hypertables.md

```diff
@@ -35,8 +35,7 @@ hypertable. To set up multi-node, see the
 
 1.  Convert the table to a distributed hypertable. Specify the name of the table
     you want to convert, the column that holds its time values, and a
-    space-partitioning parameter. For more information about space partitions,
-    see the [space-partitioning section][space-partitions].
+    space-partitioning parameter.
```

Comment:
I would keep the link here because this is about distributed hypertables so it makes sense to learn about space partitions.

Reply:
This link is to the section that was removed in the about hyper table.